### PR TITLE
{lib}[GCC/11.3.0,GCCcore/11.3.0,foss/2022a] OpenSceneGraph v3.6.5, librsvg v2.55.1, poppler v22.12.0

### PR DESCRIPTION
**Pull request by furstj**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by smoors** [comment]
@boegelbot: please test @ generoso

**Comment by smoors** [comment]
@boegelbot please test @ jsc-zen2

File: easybuild/easyconfigs/o/OpenSceneGraph/OpenSceneGraph-3.6.5-foss-2022a.eb [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+source_urls = [GITHUB_SOURCE]
+sources = ['OpenSceneGraph-%(version)s.tar.gz']
+checksums = ['aea196550f02974d6d09291c5d83b51ca6a03b3767e234a8c0e21322927d1e12']
+
+builddependencies = [
+    ('CMake', '3.23.1'),
+    ('pkg-config', '0.29.2'),
```
Comment (jfgrimm) [inline easybuild/easyconfigs/o/OpenSceneGraph/OpenSceneGraph-3.6.5-foss-2022a.eb:29]:
```suggestion
    ('pkgconf', '1.8.0'),
```

Comment (jfgrimm) [reply easybuild/easyconfigs/o/OpenSceneGraph/OpenSceneGraph-3.6.5-foss-2022a.eb:29]: `pkg-config` is no longer maintained, so for `2022a` onwards we prefer to use `pkgconf` where possible

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**FAILED**
Build succeeded for 20 out of 22 (3 easyconfigs in total)
node063.pri.viking.alces.network - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6138 CPU @ 2.00GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/fabc6dc37469f0922c7eda9bcddcd7b1 for a full test report.

**Comment by furstj** [comment]
@jfgrimm It seems that DCMTK-3.6.7 requires OpenSSL, see DCMTK-3.6.6-GCCcore-11.2.0.eb. Should I add OpenSSL enabled DCMTK to this PR or should I rather create new PR?

**Comment by smoors** [comment]
> @jfgrimm It seems that DCMTK-3.6.7 requires OpenSSL, see DCMTK-3.6.6-GCCcore-11.2.0.eb. Should I add OpenSSL enabled DCMTK to this PR or should I rather create new PR?

better to create a new PR, although it doesn't matter that much

**Comment by furstj** [comment]
@jfgrimm The DCMTK with OpenSSL has been already merged into develop. Is it possible to restart tests of #16948?

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
node064.pri.viking.alces.network - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6138 CPU @ 2.00GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/7b859b1f392ad0cd662c51488ccb6dad for a full test report.

**Comment by smoors** [comment]
Going in, thanks @furstj!

**Pull request merged**